### PR TITLE
Add citizens assemblies

### PR DESCRIPTION
**Pull request by ajparsons**
This PR adds citizens assemblies on climate to the data displayed in CAPE.

Importing from https://github.com/mysociety/citizen-assembly-data, it adds:

* Citizens assembly reports as a new document type that is displayed.
* A new council card for councils with an assembly to show just that data type and some explanatory text.
* A new filter to the list view that lets you search for councils that have had a climate assembly.
* Amended the search to allow searching by document type (and so limit a search just to assembly documents).
* A new top-level page to explain assemblies, and link to the list and search views.

Management commands to run:

```
script/manage import_citizens_assemblies
script/manage rebuild_index --noinput
script/manage import_related_searches --all
```

This gets in the new data, rebuilds the index, and redoes the related search with a slight tightening of strictness to improve relevance. 

image

image

image

image

image

### CODE REVIEW

**Review by struan — APPROVED** [review]
Looks good.

File: proj/settings.py [hunk 1]
```diff
@@ -155,7 +155,7 @@
 DECLARATIONS_CSV = os.path.join(DATA_DIR, DECLARATIONS_CSV_NAME)
 
 # The threshold for a keyphrase to be considered similar to another (0 to 1)
-RELATED_SEARCH_THRESHOLD = 0.6
+RELATED_SEARCH_THRESHOLD = 0.7
```
[the remark this answers is not in the document]
Comment (ajparsons) [reply proj/settings.py:158]: Yep! Will add something in the commit before I merge it.

**Review by zarino — APPROVED** [review]
Technically this all seems to work, so yay!

Thank you for adding a "has climate assembly" control to the advanced filters form at the top of `/plans` :-)

I hope you don’t mind @ajparsons but I tweaked the styling of the `/assemblies` page, to match the `/plans` and (emissions) `/projects` pages, given they’re all a similar kind of top level "landing page"-type page, if you know what I mean.

Screenshot 2023-09-13 at 15-37-57 Local Climate Assemblies - CAPE

**Comment by zarino** [comment]
PS @ajparsons I’d be inclined to squash all of these commits into one, given they’re all little bits of the one page:

- [Add new assembly top level page](https://github.com/mysociety/caps/pull/546/commits/745a7f1c4b2bac73e353e3c5a7e784bcb833f83f)
- [Add extra card for climate assemblies](https://github.com/mysociety/caps/pull/546/commits/0084e6eba0edbd6f7f6e41ea20178ffe4e083a83)
- [Add download link to citizens assembly data](https://github.com/mysociety/caps/pull/546/commits/a6d5fd25311aa8c4df81f2180fa17f2f386c54b6)
- [Add details of Scotland assembly to blurb](https://github.com/mysociety/caps/pull/546/commits/2a6f09a8d76d693b4b5e560ff56b6fcdc43d09b6)
- [[CAPE] Styling for climate assemblies homepage](https://github.com/mysociety/caps/pull/546/commits/c087c786fc77058cfb166d7a090185c0e8c9d9d6)

**Pull request merged**